src/vs/vscode.d.ts

```diff
@@ -1949,6 +1949,48 @@ declare module 'vscode' {
 	}
 
 	/**
+	 * Represents information about a programming construct that can be used to
+	 * identify and locate the construct's symbol. The identification does not have
+	 * to be unique, but it should be as unique as possible. It is up to the
+	 * language server to define the schema of this object.
+	 *
+	 * In contrast to `SymbolInformation`, `SymbolDescriptor` includes more concrete,
+	 * language-specific, metadata about the symbol.
+	 */
```

Comment:
monaco.d.ts is generated. I tried copying over the comments but they just get overwritten on next generation, but not sure why they don't get picked up.